### PR TITLE
bug fixes post kaimerge

### PR DESCRIPTION
**Pull request by tazlin**
- Fixed a couple of cases of `{e}` not referring to a variable (missing `Exception as e`)
- Fixed a few cases of missing `self.` for variables or a `self` arg in a class method
- Removed a duplicated dictionary entry in `models\v2.py`.
- Fixed adding a team, which currently broken (500 and it doesn't work) due to a missing import, probably missed because it would have required a circular reference (`database\functions.py` logically required `base\team.py` and vice versa).

### CODE REVIEW

File: horde/model_reference.py [hunk 1]
```diff
@@ -40,11 +40,11 @@ def call_function(self):
                 if self.text_reference[model].get("nsfw"):
                     self.nsfw_models.add(model)
 
-        except Exception:
+        except Exception as e:
             logger.error(f"Error when downloading known models list: {e}")
 
     def get_model_names(self):
-        return set(reference.keys())
+        return set(self.reference.keys())
```
Comment (tazlin) [inline horde/model_reference.py:47]: @db0 is this related to the model issues you're having?

Comment (db0) [reply horde/model_reference.py:47]: No that was a caching issue

Comment (db0) [reply horde/model_reference.py:47]: I don't actuallyt use this method afaics
